internal/machines/machines_test.go

```diff
@@ -956,7 +956,7 @@ func TestCreateSystemSnapshot(t *testing.T) {
 		"Error on existing snapshot on user child":   {def: "m_with_userdata_and_multiple_snapshots.yaml", snapshotName: "user_child_snapshot", wantErr: true, isNoOp: true},
 
 		"Error when name starts with dash":            {def: "m_with_userdata.yaml", snapshotName: "-my_snapshot", wantErr: true, isNoOp: true},
-		"Error when name contains invalid characters": {def: "m_with_userdata.yaml", snapshotName: "my snäpshôt,", wantErr: true, isNoOp: true},
+		"Error when name contains invalid characters": {def: "m_with_userdata.yaml", snapshotName: "my, snäpshôt, is beautiful,", wantErr: true, isNoOp: true},
```

Comment:
I would have kept the original test and add another one for the deduplication itself.

Reply:
As discussed, we can’t get the output, so the tests will be duplicated. Let’s merge it.